pages/handbook/product-engineering/how-we-work/workflow.mdx

```diff
@@ -0,0 +1,110 @@
+# Engineering Workflow
+
+## Specification
+
+Goal: we should make sure to have as few loops as possible per change.
+
+- Small/mid sized changes (e.g. adding a new filter to a table, adding a new field to a form…)
+  - Engineer leads, very brief notes on linear or just a descriptive title
+  - Open to do brief discussions with Max/Marc if helpful to speed process up and reduce risk/uncertainty
+  - If you plan to have a reviewer of a change, please make sure to involve the reviewer in the planning process.
+- Large projects (e.g. supporting Agents in Langfuse, rebuilding SDKs ..)
+  1. Engineer does initial investigation and then schedules meeting with Marc/Max and other team members who have a lot of relevant context
+  2. Meeting goal: Derisking and clarifying all important topics
+     - Recorded \-\> meeting includes lots of details, good to generate spec/issue-description/working with Claude Code
+     - Discussion on timelines and how we can make cut requirements to build faster
+     - Implementation plan afterwards, spin with Max if needed
+     - If things change in a meeting (also follow-up discussions), small note logged to Linear ticket / project.
+     - Engineer needs to make sure to have relevant people in the room to make a decision ([DRI](https://docs.google.com/spreadsheets/d/1gOvWf_uSAtcXxWkR_gMuWX8-OYmSJNIPXTmPGfZ2DGY/edit?gid=0#gid=0)).
+     - Engineer needs to manage Linear project based on the outcome of the meeting.
+  3. Engineer may need input on UI/UX or Clickhouse queries
+     - Get quick feedback on implementation thoughts from respective owners ([DRI](https://docs.google.com/spreadsheets/d/1gOvWf_uSAtcXxWkR_gMuWX8-OYmSJNIPXTmPGfZ2DGY/edit?gid=0#gid=0)).
+     - Based on the initial discussion, ask the owner for PR reviews if needed.
+
+## Prioritization
+
+In case of uncertainty: Tag Max (24h SLA on Linear inbox). Everyone should have a clear view on what priorities are. The best way to achieve this is by having one Linear project or issue for the current main task. For bugs and smaller improvements, use the [bug view](https://linear.app/langfuse/view/bugs-confirmedopen-97856f9f745c) or “My issues”. Tickets in Linear should always have a prioritization:
+
+- **State Triage**
+  - State: unassigned, no labels or prios
+  - Created via
+    - GitHub Issue integration
+    - In Linear by non-engineering team members
+  - Marc/Max subscribe, triage and assign, and add labels, and refine title
+  - After assignment, Engineer dedupes with existing tickets and handles communications with users.
+- **State Backlog**
+  - Everyone manages their backlogs.
```

Comment:
"Everyone manages their own backlog" is something I would pull up and highlight.

- Everyone is resposnible for their linear issues (no central management)
- We try to align around a common understanding of priorities and SLAs
- Every IC has the resonsibility to prioritize and escalate if they cannot do so on their own